src/Symfony/Component/Messenger/MessageBusInterface.php

```diff
@@ -19,11 +19,7 @@
     /**
      * Dispatches the given message.
      *
-     * The bus can return a value coming from handlers, but is not required to do so.
-     *
      * @param object|Envelope $message The message or the message pre-wrapped in an envelope
```

Comment:
What about adding phpdoc for the new argument? Describe when this name can be useful